openpype/modules/deadline/plugins/publish/submit_maya_deadline.py

```diff
@@ -503,7 +503,13 @@ def process(self, instance):
 
         self.payload_skeleton["JobInfo"]["Comment"] = comment
         self.payload_skeleton["PluginInfo"]["RenderLayer"] = renderlayer
-        self.payload_skeleton["PluginInfo"]["RenderSetupIncludeLights"] = get_current_project_settings()["maya"]["create"]["CreateRender"]["enable_all_lights"] # noqa
+        self.payload_skeleton["PluginInfo"]["RenderSetupIncludeLights"] = (
+            context.data["project_settings"].get(
+                "maya", {}).get(
+                "create", {}).get(
+                "CreateRender", {}).get(
+                "enable_all_lights", {})
+        )
```

Comment:
A bit off topic.

Anyone else feels like we should have a library function that allows to quickly to basically do this nested retrieval and fall back on a default value in a more readable way:

```python
get(settings, "maya.create.CreateRender.enable_all_lights", default=None)
get(settings, "maya", "create", "CreateRender", "enable_all_lights", default=None)
```

Reply:
>Anyone else feels like we should have a library function that allows to quickly to basically do this nested retrieval and fall back on a default value in a more readable way:

Topic for another PR, but in general I kinda agree.